accepted/build/build-publish-tweaks.md

```diff
@@ -0,0 +1,28 @@
+# Build and Publish tweaks in the .NET Core SDK
+
+In .NET Core 3, we are making Build output more similar to the Publish output.  In .NET Core 2.x and prior, dependencies from NuGet packages are not copied into the Build output.  At runtime, they are loaded from the NuGet package cache.  This meant that the Build output was not portable to other machines.
```

Comment:
> This meant that the Build output was not portable to other machines.

If making the Build output portable to other machines is a goal, are we also no longer writing `additionalProbingPaths` in the `.runtimeconfig.dev.json` file?

```json
{
  "runtimeOptions": {
    "additionalProbingPaths": [
      "C:\\Users\\eerhardt\\.dotnet\\store\\|arch|\\|tfm|",
      "C:\\Users\\eerhardt\\.nuget\\packages",
      "C:\\Program Files\\dotnet\\sdk\\NuGetFallbackFolder"
    ]
  }
}
```

Reply:
We are still writing it, but I think it needs to be conditioned on something. I mentioned this elsewhere here, I think. I would say the last two should be conditioned on CopyLocalLockFileAssemblies being false and the first one would require a new opt in like UseLocalRuntimeStore (breaking, though we are making other breaks to dotnet store right now.)

Reply:
See also https://github.com/dotnet/designs/pull/78#discussion_r289151178